test/vestingscheme.js

```diff
@@ -219,7 +249,7 @@ contract('VestingScheme', function(accounts) {
                assert.equal(await testSetup.org.token.balanceOf(testSetup.vestingScheme.address),amountPerPeriod*numberOfAgreedPeriods);
               });
 
-              it("createVestedAgreement check agreement id ", async function() {
+              it("createVestedAgreement check agreement", async function() {
```

Comment:
this test check the agreement id .why to remove it ?

Reply:
I put it back